src/data/location-sources/custom-azure-zones.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Should we add docs around this file somewhere?

Reply:
Will make this a template file so that this won't be picked up by config
@juzuluag 
- Add a story for overriding and how to manage multiple files etc...
- Make a template
- Update Docs

Reply:
@JenMadiedo , all resolved with this file.